extensions/wikia/Recirculation/services/DiscussionsDataService.class.php

```diff
@@ -83,7 +83,7 @@ private function apiRequest() {
 		$endpoint = $this->cityId . '/threads';
 
 		$url = $this->buildUrl( $endpoint, $options );
-		$data = Http::get( $url );
+		$data = Http::get( $url, 'default', [ 'noProxy' => true ] );
```

Comment:
`noProxy` defaults to false, which causes the request to be proxied through `prod.border.service.consul:80`, which would then fail. So we need to set `noProxy` to `true` here.